hack/godeps/godep-restore.sh

```diff
@@ -33,6 +33,7 @@ if [ ! -d "${KUBE_ROOT}" ]; then
 fi
 
 godep::restore_kubernetes
+godep::remove_staging_from_json
```

Comment:
Ideally, a "restore" shouldn't make your repo dirty.  

However, I understand this will break if you have some of the staging libraries cloned from their bot-pushed downstream repos.  We have to do this since localkube vendors kubernetes code which references the staging/ libraries directly, rather than the split out repos - and the downstream repos are not synced regularly enough.  The contributing doc should note this.

If you remove the staging entries from the godep json, you won't get the right versions of the  staging libraries.

Reply:
So as-is this script will not work with a clean checkout and new `GOPATH` following the instructions. What is the suggested approach?